README_PYPI.rst

```diff
@@ -1,25 +1,48 @@
 .. # README file for Pypi
 
-Pywbemtools is a collection of command line tools  is a WBEM client,
-written in pure Python. It supports Python 2 and Python 3.
+Pywbemtools is a collection of command line tools that communicate with WBEM
+Servers. The tools are written in pure Python and support Python 2 and Python
+3.
 
-A WBEM client allows issuing WBEM operations to a WBEM server, using the
-`CIM/WBEM standards`_ defined by the DMTF, for the purpose of performing
-systems management tasks. A WBEM indication listener is used to wait for
-and process notifications emitted by a WBEM server, also for the purpose
-of systems management.
+Pywbemtools provides a command line tool (pywbemcli) that uses the `pywbem`_
+Python WBEM client infrastructure to issue operations to a WBEM server using
+the `CIM/WBEM standards`_ defined by the `DMTF`_ to perform system management
+tasks.
 
-In addition to providing a command line interface for the client side DMTF defined
-WBEM Operations, pywbem cli provides commands for a number of high level
-operations that are useful to developers and users of WBEM Servers such
-as the implementation of the SNIA SMI specification.
+CIM/WBEM infrastructure is used for a wide variety of systems management tasks
+in the industry including DMTF management standards and  the `SNIA`_
+Storage Management Initiative Specification `SMI-S`_.
 
-CIM/WBEM infrastructure is used for a wide variety of systems management
-tasks in the industry.
+Pywbemcli provides a multilevel command line syntax to communicate with a WBEM
+server to:
+
+      * Explore the CIM data of WBEM Server.  It can manage/inspect the CIM
+        model components including CIM_Classes, CIM_instances, and CIM
+        qualifiers and execute CIM methods and queries on the WBEM server. It
+        implements subcommands to execute almost all of the defined DMTF
+        CIM/XML operations.
+
+      * Inspect WBEM Server components such as namespaces, registered
+        profiles, and other server information.
+
+      * Capture detailed information on the interactions with the WBEM server
+        including time statistics.
+
+For more information on pywbemtools see the `pywbemtools readme`_ and the
+`pywbemtools documentation`_
+
+.. _pywbemtools readme: https://github.com/pywbem/pywbemtools/blob/stable_0.6/README.rst
+.. _pywbemtools documentation: https://pywbemtools.readthedocs.io/en/stable_0.6/
```

Comment:
Referencing the readme and docs using the `stable_0.6` branch requires updating the version in this file upon every release. I suggest to simply reference the latest version (which is master, i.e. not released).

Reply:
Done.